cipher_suite_tls_ecdhe_ecdsa_with_aes_128_gcm_sha256.go

```diff
@@ -30,6 +31,10 @@ func (c cipherSuiteTLSEcdheEcdsaWithAes128GcmSha256) isPSK() bool {
 	return false
 }
 
+func (c cipherSuiteTLSEcdheEcdsaWithAes128GcmSha256) isInitialized() bool {
+	return c.initialized
```

Comment:
Would it be possible to do `return c.gcm != nil` ?

Reply:
Yes, good idea!